Trimble.Modus.Components/Controls/TMDataGrid/ColumnTypes/TextColumn.xaml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<modus:DataGridColumn xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                      xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                      xmlns:modus="clr-namespace:Trimble.Modus.Components"
+                      x:Class="Trimble.Modus.Components.TextColumn">
+  <modus:DataGridColumn.CellTemplate>
+    <DataTemplate>
+      <Grid RowSpacing="2">
+        <Grid.RowDefinitions>
+          <RowDefinition Height="*" />
+          <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <Label Grid.Row="0"
```

Comment:
I don't see any bindings or x:Name or anything here. Can you explain how this works?

Reply:
The binding is set in `DataGridRow.CreateCell()` method.